File: Examples/DateToolsExample/DateToolsExample/ViewController.m
//
//  ViewController.m
//  DateToolsExample
//
//  Created by Matthew York on 3/19/14.
//
//

#import "ViewController.h"
#import "NSDate+DateTools.h"
#import "DTTimePeriodCollection.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
    
    DTTimePeriodCollection *collection = [DTTimePeriodCollection collection];
    [collection insertTimePeriod:[DTTimePeriod timePeriodWithAllTime] atIndex:0];
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Tests/DateToolsTests/DateToolsTests/ViewController.swift
//
//  ViewController.swift
//  DateToolsTests
//
//  Created by Matthew York on 8/19/16.
//  Copyright © 2016 Matthew York. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        let formatter = DateFormatter()
        formatter.dateFormat = "yyyy MM dd HH:mm:ss.SSS"
        let date = formatter.date(from: "2015 4 05 18:15:12.000")!
        print(Date().chunkBetween(date: date))
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
}

